src/java/org/apache/cassandra/utils/bytecomparable/ByteSourceInverse.java

```diff
@@ -402,6 +402,19 @@ public static byte[] readBytes(ByteSource byteSource)
         return readBytes(byteSource, INITIAL_BUFFER_CAPACITY);
     }
 
+    public static void copyBytes(ByteSource byteSource, byte[] bytes)
```

Comment:
I have no idea what qualifies a utility for inclusion in `ByteSourceInverse` vs. `ByteSource` or somewhere else...just double-checking to see if this is the right home...